### PR TITLE
[CCAP-355] Discourage providers from declining care

### PR DESCRIPTION
**Pull request by spokenbird**
#### 🔗 Jira ticket
CCAP-355

#### ✍️ Description
Adds a confirmation screen when provider declines to provide care.

#### 📷 Design reference
https://www.figma.com/design/1oRFY7p6tdGogBJk7Ugle8/IL-CCAP-Families-Full-Flow?node-id=9983-25668&t=E36qtaE8FbyiBlAi-4

### CODE REVIEW

File: src/main/java/org/ilgcc/app/utils/ProviderSubmissionUtilities.java [hunk 1]
```diff
@@ -70,6 +70,27 @@ public static List<Map<String, String>> getChildrenDataForProviderResponse(Submi
         }
         return children;
     }
+    
+    public static String formatChildNamesAsComaSeperatedList(Submission applicantSubmission) {
```
Comment (cram-cfa) [inline src/main/java/org/ilgcc/app/utils/ProviderSubmissionUtilities.java:74]: You're missing an `m` here and also `Separated` not `Seperated`. Should be `commaSeparatedList`

File: src/test/java/org/ilgcc/app/utils/ProviderSubmissionUtilitiesTest.java [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+import static org.junit.jupiter.api.Assertions.*;
+
+import formflow.library.data.Submission;
+import org.junit.jupiter.api.Test;
+
+class ProviderSubmissionUtilitiesTest {
+
+    @Test
+    void formatChildNamesAsComaSeperatedList() {
```
Comment (cram-cfa) [inline src/test/java/org/ilgcc/app/utils/ProviderSubmissionUtilitiesTest.java:12]: Same on the spelling.

**Review by cram-cfa — CHANGES_REQUESTED** [review]
Looks good other than the spelling of that method name and the related test case!

Thanks for including tests!!

**Review by cram-cfa — APPROVED** [review]

**Review by analoo — APPROVED** [review]

**Comment by analoo** [comment]
I appreciate the tests and thee child name helper method that lists out the names of children in a grammatically correct way.

**Pull request merged**